src/components/Head.js

```diff
@@ -0,0 +1,46 @@
+import NextHead from 'next/head';
+import { useRouter } from 'next/router';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Head = ({
+  title, description, ogDescription, ogImage, ogTitle,
+}) => {
+  const router = useRouter();
+  return (
+    <NextHead>
+      <title>
+        {title}
+        {' '}
+        | YSM
+      </title>
+      <meta name="description" content={description} />
+      <meta property="og:url" content={`${process.env.BASE_URL} ${router.pathname}`} />
+      <meta property="og:type" content="website" />
+      <meta property="og:description" content={ogDescription || description} />
+      <meta property="og:title" content={ogTitle || title} />
+      <meta property="og:image" content={ogImage} />
```

Comment:
What happens if `ogImage` is not set / is blank?

Reply:
Have set logo.png as a backup, but if it was blank google/fb/twitter would choose an image from inside the page.